pkg/codegen/docs/templates/package_details.tmpl

```diff
@@ -9,5 +9,9 @@
 	<dt>Notes</dt>
 	<dd>{{ htmlSafe .Notes }}</dd>
     {{- end }}
+	{{ if ne .Version "" -}}
```

Comment:
The indentation looks a bit off here? Could you please take a look?

Reply:
yea, it was really weird. I also noticed this when I was initially reviewing the diffs. On my machine it was showing up as being perfectly aligned when opened in VSCode, must be some weird white space issue. Let me fool with it and see if it will cooperate.

Reply:
OK. I think it's correct now. Appears one line was using tabs and the other had spaces.... lol. 🙄

Reply:
Ahh I see. Thank you for fixing it :)